cmd/config/config.go

```diff
@@ -42,7 +43,7 @@ func main() {
 		key = string(result)
 	}
 
```

Comment:
Unrelated to this line, but as discussed, os.Rename impacts Docker. Can replace usage with a tested version of https://gist.github.com/var23rav/23ae5d0d4d830aff886c3c970b8f6c6b

Reply:
Good find, have submitted a PR for this!